metadata-ingestion/src/datahub/sdk/datajob.py

```diff
@@ -68,11 +62,9 @@ def __init__(
         self,
         *,
         # Identity
-        flow_urn: DataflowUrnOrStr,
+        flow: DataFlow,
```

Comment:
we should be supporting either one. e.g. either you pass in `flow_urn + platform_instance`, or you can pass in `flow`.

Reply:
using `typing.overload`, and with appropriate error-checking / good error messages in the init method

Reply:
updated -- FYI i didn't used overload, let me know if there's a strong reason that we should use this